tests/report_test.sh

```diff
@@ -117,12 +117,12 @@ function test_grouping_day_data()
   local line
 
   declare -a expected_content=(
-    ' * [06:00:40-06:20:40][20m]: Tag 1 description\n'
-    ' * [08:30:50-08:45:50][15m]: Tag 2 description\n'
-    ' * [09:00:00-10:00:00][1h]: Tag 3 description\n'
-    ' * [11:00:00-11:00:44][44s]: Tag 4 description\n'
-    ' * [14:00:00-14:30:00][30m]: Tag 5 description\n'
-    ' * [15:00:00-15:10:00][10m]\n'
+    $' * [06:00:40-06:20:40][20m]: Tag 1 description\n'
+    $' * [08:30:50-08:45:50][15m]: Tag 2 description\n'
+    $' * [09:00:00-10:00:00][1h]: Tag 3 description\n'
+    $' * [11:00:00-11:00:44][44s]: Tag 4 description\n'
+    $' * [14:00:00-14:30:00][30m]: Tag 5 description\n'
+    $' * [15:00:00-15:10:00][10m]\n'
```

Comment:
At first glance, this `$` in the beginning looks a little bit confusing... Could you just add a single comment before `expected_content`?